### PR TITLE
feat: bria chart

### PR DESCRIPTION
**Pull request by sandipndev**
Get https://github.com/GaloyMoney/bria into production.

### CODE REVIEW

File: charts/bria/templates/bria-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "bria.fullname" . }}-config
+data:
+  bria.yml: |-
```
Comment (bodymindarts) [inline charts/bria/templates/bria-cm.yaml:6]: this won't work. Need a translation from `camelCase` <- idiomatic helm to `snake_case` <- expected in bria.
See how this is done in stablesats.

File: charts/bria/templates/bria-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        - bria
+        - daemon
+        volumeMounts:
+        - name: "config"
+          mountPath: "/bria.yml"
+          subPath: "bria.yml"
+        env:
+        - name: BRIA_CONFIG
+          value: "/bria.yml"
+        - name: BITCOIND_HOST
```
Comment (bodymindarts) [inline charts/bria/templates/bria-deployment.yaml:44]: These `XXX_HOST` values should be injected into config - reserve env vars for secrets.

**Review by bodymindarts — CHANGES_REQUESTED** [review]

File: charts/bria/templates/bria-deployment.yaml (hunk 2 above)
Comment (sandipndev) [reply charts/bria/templates/bria-deployment.yaml:44]: this would require source code changes: config needs to be `EnvOverride`den.

Comment (sandipndev) [reply charts/bria/templates/bria-deployment.yaml]: done

File: charts/bria/templates/bria-cm.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    migrate_on_start: true
+    blockchain:
+      network: {{ .Values.bria.blockchain.network }}
+      electrum_url: {{ .Values.bria.blockchain.electrumUrl }}
+    wallets:
```
Comment (bodymindarts) [inline charts/bria/templates/bria-cm.yaml:11]: Don't add these - lets use defaults everywhere until we need to change them.

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**